README.md

```diff
@@ -5,6 +5,8 @@ Scientific computing can be difficult in practice due to various complex softwar
 ![BPS Poster](BPS_poster.svg)
 
 ## Documentation
+![buid-status](https://readthedocs.org/projects/workflow-inference-compiler/badge/?version=latest&style=svg)
```

Comment:
Can we move this to the very top? This is the common practice for using this type for status links (for quick glance). See `Nyxus`/`bfio`.